modules/blog/post-snippet/post-snippet.tsx

```diff
@@ -166,9 +165,13 @@ const PostSnippet = ({
         <Image
           alt={post.coverImage.title || 'Generic image for blog post'}
           height={imgSize}
-          objectFit="cover"
           src={post.coverImage.url}
           width={imgSize}
+          style={{
+            maxWidth: '100%',
+            height: 'auto',
+            objectFit: 'cover',
```

Comment:
Nit: a thing that I like to do (and try to get the team to do) is to sort styles and props alphabetically. The more there are, the easier it is to read.

So in this instance, it would be:

```suggestion
            height: 'auto',
            maxWidth: '100%',
            objectFit: 'cover',
```

Reply:
Done.

Reply:
Thank you! Merge this whenever you can :)